.travis.yml

```diff
@@ -45,3 +45,6 @@ branches:
   except:
     - staging
     - production
+
+addons:
+  chrome: stable
```

Comment:
I did this to see if it would install dependencies which Cypress also needed - this can be removed as it didn't help.